packages/headless/src/controllers/core/search-parameter-manager/headless-core-search-parameter-manager.test.ts

```diff
@@ -28,7 +28,6 @@ import {
   buildCoreSearchParameterManager,
```

Comment:
We should also add the following test:

- when #initialState.parameters.tab is defined, dispatches #restoreTab with the correct argument before #restoreSearchParameters on registration

and add something similar under describe('#synchronize').

We should also adjust the 'dispatches #restoreSearchParameters on registration' test to ensure that the action is dispatched with the correct argument (i.e., the params without the tab), and adjust the existing tests under the #sychronize describe accordingly as well.